tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -120,7 +120,19 @@
 
 ORA2_FILEUPLOAD_BACKEND = "filesystem"
 ORA2_FILEUPLOAD_ROOT = "/openedx/data/ora2"
+FILE_UPLOAD_STORAGE_BUCKET_NAME = "openedxuploads"
 ORA2_FILEUPLOAD_CACHE_NAME = "ora2-storage"
+# For a long while, we were storing ora2 uploads this directory.
+ora2_folder = os.path.join(ORA2_FILEUPLOAD_ROOT, FILE_UPLOAD_STORAGE_BUCKET_NAME)
+deprecated_ora2_folder = os.path.join(ORA2_FILEUPLOAD_ROOT, "SET-ME-PLEASE (ex. bucket-name)")
+if deprecated_ora2_folder != ora2_folder and os.path.exists(deprecated_ora2_folder):
```

Comment:
```suggestion
if os.path.exists(deprecated_ora2_folder):
```
This:
> deprecated_ora2_folder != ora2_folder

 will always be true, as the former will always equal `$ORA2_FILEUPLOAD_ROOT/SET-ME-PLEASE (ex. bucket-name)` and the latter `$ORA2_FILEUPLOAD_ROOT/openedxuploads`.